internal/browsir.go

```diff
@@ -21,7 +21,16 @@ type ICommand interface {
 	preview(args []string)
 }
 
-type Command struct{}
+type Command struct {
+	linkLoader LinkLoader
```

Comment:
I think LinkLoader and RealLinkLoader are not defined anywhere, you should add the interface somewhere!